app/models/work_activity.rb

```diff
@@ -77,7 +98,7 @@ def provenance_notes_type?
   end
 
   def log_event_type?
-    system_event_type? || file_changes_event_type? || changes_event_type? || provenance_notes_type?
+    CHANGE_LOG_ACTIVITY_TYPES.include? activity_type
```

Comment:
Are the methods above still used? Maybe `def changes_event_type?` and the rest can be removed.

Reply:
I believe they are utilized for the HTML formatting, but I did not check them all.